enhancements/update/versioning-cincinnati-api.md

```diff
@@ -0,0 +1,171 @@
+---
+title: versioning-cincinnati-api
+authors:
+- "@pratikmahajan"
+  reviewers:
+- @openshift/openshift-team-cincinnati-maintainers
+- @openshift/openshift-team-cincinnati
+- @openshift/team-cincinnati-operator
```

Comment:
nit: teams evolve, so before this lands its probably best to bump it to match the set of folks who have weighed in so far.

Reply:
changed reviewers to add specific people